README.md

```diff
@@ -253,9 +254,62 @@ Map<String,String> queryParams = new HashMap<String,String>(){{
 CategoryResponse getCategories = butterClient.getCategory("java",queryParams);
 ```
 
+## Tags
+
+### List Tags
+
+Listing Tags returns a [TagsResponse](#tagsresponse-class) object. This object consists of a List&lt;[Tag](#Tag-class)&gt;
```

Comment:
Out of curiosity, what will happen if you use `<` and `>` directly instead of `&lt;` and `&gt;` in that Markdown file? Will it treat that as an inline HTML code?

Reply:
Sometimes it does and sometimes it doesn't - like the behavior is inconsistent.

Reply:
Ok, let's leave the escaped form, then.